docs/usage/files.md

```diff
@@ -217,6 +217,28 @@ print('File "{0}" uploaded to Box with file ID {1}'.format(uploaded_file.name, u
 [get_chunked_uploader]: https://box-python-sdk.readthedocs.io/en/latest/boxsdk.object.html#boxsdk.object.upload_session.UploadSession.get_chunked_uploader
 [get_chunked_uploader_for_stream]: https://box-python-sdk.readthedocs.io/en/latest/boxsdk.object.html#boxsdk.object.upload_session.UploadSession.get_chunked_uploader_for_stream
 
+#### Resume Upload
+
+Sometimes an upload can be interrupted, in order to resume uploading where you last left off, simply call the
+[`chunked_uploader.resume()`][resume] method. This will return the the [File][file_class] object that was uploaded.
+
+```python
+chunked_uploader = client.file('12345').get_chunked_uploader('/path/to/file')
+uploaded_file = chunked_uploader.resume()
+print('File "{0}" uploaded to Box with file ID {1}'.format(uploaded_file.name, uploaded_file.id))
+```
+
+Alternatively, you can also create a [`UploadSession`][upload_session_class] object by calling
```

Comment:
It might be good to explain exactly why the user would want to do this (i.e. in order to resume an existing upload session)